blueman/gui/manager/ManagerDeviceList.py

```diff
@@ -200,14 +204,20 @@ def make_device_icon(self, icon_info: Gtk.IconInfo, is_paired: bool = False, is_
         ctx = cairo.Context(target)
 
         if is_paired:
-            icon_info = self.get_icon_info("dialog-password", 16, False)
+            _icon_info = self.get_icon_info("dialog-password", 16, False)
+            assert _icon_info is not None
+            icon_info = _icon_info
             paired_surface = icon_info.load_surface(window)
             ctx.set_source_surface(paired_surface, 1 / scale, 1 / scale)
             ctx.paint_with_alpha(0.8)
 
         if is_trusted:
-            icon_info = self.get_icon_info("blueman-trust", 16, False)
+            _icon_info = self.get_icon_info("blueman-trust", 16, False)
+            assert _icon_info is not None
+            icon_info = _icon_info
             trusted_surface = icon_info.load_surface(window)
+            assert isinstance(target, cairo.ImageSurface)
+            assert isinstance(trusted_surface, cairo.ImageSurface)
```

Comment:
This should be fixed in the stub that load_surface return an instance of cairo.Surface. Or just force it in the stub to be an cairo.ImageSurface as I doubt we'll ever see anything else.

Reply:
> This should be fixed in the stub that load_surface return an instance of cairo.Surface.
What do you mean? I think that's the case there, but we expect that `Surface` to be an `ImageSurface` which is not guaranteed.